controllers/replication.storage/volumereplication_controller.go

```diff
@@ -380,6 +380,15 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 			if ts != nil {
 				lastSyncTime := metav1.NewTime(ts.AsTime())
 				instance.Status.LastSyncTime = &lastSyncTime
+				td := info.GetLastSyncDuration()
+				if td != nil {
+					lastSyncDuration := metav1.Duration{Duration: td.AsDuration()}
+					instance.Status.LastSyncDuration = &lastSyncDuration
+				}
+				tb := info.GetLastSyncBytes()
+				if tb != 0 {
+					instance.Status.LastSyncBytes = &tb
+				}
```

Comment:
Fields should be explicitly set to nil to indicate missing value.
otherwise, they may inherit values that were previously set.

Reply:
I am not sure about this, considering the corner case where lastsynctime retains its value and on requeue other values will be nil. Will verify this once and update if it's fine.

Reply:
On line 91 (wow, what a horrible long function this is?!) `instance` is fetched, so it will have a `.Status.LastSyncBytes` from the previous reconcile. If there were 0 bytes in the last sync, the information would not be correct in the object. So yes, setting it explicitly to an initial value is probably a good thing.

Reply:
yeah done

Reply:
@Rakshith-R @nixpanic IMO we should not set this to nil explicitly https://github.com/csi-addons/kubernetes-csi-addons/pull/366#discussion_r1238259918